server/reports/expiring-items/1_0/manifest.json

```
@@ -0,0 +1,16 @@
+{
+    "isCustom": false,
+    "version": "2.3.1",
```

```suggestion
    "version": "2.3.0",
```

??? this will be the first release of `2.3` right? Or do we start with `.1`?

---

Start with .0